source/expressions.tex

```diff
@@ -1947,23 +1947,22 @@
 
 \begin{bnf}
 \nontermdef{capture}\br
-    simple-capture \opt{\terminal{...}}\br
-    % FIXME: This is wrong. The ... should go after the &.
-    \opt{\terminal{...}} init-capture
+    simple-capture\br
+    init-capture
 \end{bnf}
 
 \begin{bnf}
 \nontermdef{simple-capture}\br
-    identifier\br
-    \terminal{\&} identifier\br
+    identifier \opt{\terminal{...}}\br
+    \terminal{\&} identifier \opt{\terminal{...}}\br
```

Comment:
Wow, I am... just... absolutely the worst at this. I am so sorry.

This PR matches the paper. The paper is incorrect. The `...` should be between the `&` and the `identifier`, as they are in the init-capture case. Can we just Do The Right Thing ™️ here?

@zygoloid @jensmaurer